<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWin</class>
 <widget class="QWidget" name="LoginWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWin</string>
  </property>
  <widget class="QLabel" name="lbName">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leName">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbPassword">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>480</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
